puppet/modules/kubernetes/templates/kube-proxy.service.erb

```diff
@@ -9,12 +9,16 @@ ExecStartPre=/sbin/sysctl -w net.bridge.bridge-nf-call-iptables=1
 ExecStartPre=/sbin/sysctl -w net.bridge.bridge-nf-call-ip6tables=1
 ExecStart=<%= scope['kubernetes::_dest_dir'] %>/proxy \
   --v=<%= scope['kubernetes::log_level'] %> \
+  --logtostderr=true \
+<% if @post_1_11 -%>
+  --config=<%= @proxy_config_file  %> \
+<% else -%>
   --cluster-cidr=<%= scope['kubernetes::pod_network'] %> \
   --resource-container=podruntime.slice \
 <% if @kubeconfig_path -%>
   --kubeconfig=<%= @kubeconfig_path %> \
 <% end -%>
-  --logtostderr=true
```

Comment:
We always need to end with no back slash. Can you make this flag the last parameter again